packages/features/calendars/weeklyview/components/event/Empty.tsx

```diff
@@ -10,19 +12,11 @@ import type { GridCellToDateProps } from "../../utils";
 import { gridCellToDateTime } from "../../utils";
 
 type EmptyCellProps = GridCellToDateProps & {
-  availableSlots?: CalendarAvailableTimeslots;
+  isDisabled?: boolean;
+  topOffsetMinutes?: number;
 };
 
 export function EmptyCell(props: EmptyCellProps) {
-  const timeFormat = useTimePreferences((state) => state.timeFormat);
```

Comment:
Quite a bit of logic is removed and moved down to the `Cell` component. We reuse that in `EmptyCell` as well as `AvailableCellsForToday` (also below).